.circleci/config.yml

```diff
@@ -2,6 +2,28 @@ version: 2.1
 orbs:
   elixir: membraneframework/elixir@1
 
+jobs:
+  lint_frontend:
+    working_directory: ~/membrane_live/assets
+    docker:
+      - image: cimg/node:18.4.0
```

Comment:
Things got a little bit more complicated regarding the `.tool-versions`. The docker used by the circleci to build the pipelines is [running the `membraneframework-labs/docker_membrane`](https://app.circleci.com/pipelines/github/membraneframework/membrane_videoroom/545/workflows/46b07ea6-8ae4-4e84-9977-0926fb8bc190/jobs/1529?invite=true#step-0-13) which uses version 1.13.0 of elixir and 25.0.2 of erlang. It would be necessary to change that in the docker or use the specified versions here.

Reply:
Let's leave it out of this PR, the image has to be updated